xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-rest/src/main/java/org/xwiki/livedata/internal/rest/DefaultLiveDataEntryResource.java

```diff
@@ -79,6 +79,8 @@ public Response updateEntry(String sourceId, String namespace, String entryId, E
                     Entry updatedEntry =
                         createEntry(values.get(), updatedEntryId.get(), config.getQuery().getSource(), namespace);
                     return Response.status(Status.ACCEPTED).entity(updatedEntry).build();
+                } else {
+                    return Response.status(Status.ACCEPTED).build();
```

Comment:
Why? Because the entry can be deleted by someone else between the ``save`` and the ``get`` calls? Deserves a comment at least.

Reply:
This was because `liveTable` does not implements the `get` method. I've updated the code to rely on the `UnsupportedOperationException` exception:
https://github.com/xwiki/xwiki-platform/pull/1542/files#diff-d5f0a1eac1f978ab5a56bc091e47a16d7317ce6614b9fb207c6e350407525a82R80-R84